plugins/org.jboss.tools.openshift.express.ui/src/org/jboss/tools/openshift/express/internal/ui/console/ConsoleUtils.java

```diff
@@ -48,26 +39,12 @@
 public class ConsoleUtils implements IConsoleUtility {
 
 	/**
-	 * Constant key set into the created message console attributes to mark the
-	 * given console as an 'openshift' one.
-	 */
-	public static final String CONSOLE_TYPE_KEY = "ConsoleType";
-
-	/**
-	 * Constant value set into the created message console attributes to mark
-	 * the given console as an 'openshift' one.
-	 */
-	public static final String CONSOLE_TYPE_VALUE = "OpenShiftTailConsole";
-
-	/**
 	 * Registers the given listener as a console listener.
 	 * 
 	 * @param consoleListener
 	 */
 	public static void registerConsoleListener(IConsoleListener consoleListener) {
-		ConsolePlugin plugin = ConsolePlugin.getDefault();
-		IConsoleManager consoleManager = plugin.getConsoleManager();
-		consoleManager.addConsoleListener(consoleListener);
+		org.jboss.tools.openshift.internal.common.ui.console.ConsoleUtils.registerConsoleListener(consoleListener);
```

Comment:
I used the full package to distinguish the fact that the common methods were moved out of express.